include/mirage/persistent_kernel/tasks/utils.cuh

```diff
@@ -14,6 +14,11 @@
  */
 #pragma once
 #include "common.h"
+
+#define CLEAR_8_FLOATS(v) \
```

Comment:
Can we make `clear_8_floats` an inlined function instead of a marco?
```
template<typename T>
__device__ __forceinlined__ void clear_8_floats(T * buf) {
(static_cast<__uint128_t *>(v))[0] = 0ul;
(static_cast<__uint128_t *>(v))[1] = 0ul;
}
```